test/components/Post/component.test.js

```diff
@@ -71,24 +67,22 @@ describe('Post', () => {
   it('passes community', () => {
     const props = {post: {communities: [{slug: 'passedcommunity'}]}}
     const wrapper = renderComponent(props)
-    expect(wrapper.find('Details').prop('community'))
+    expect(wrapper.find('PostDetails').prop('community'))
     .to.deep.equal(props.post.communities[0])
   })
 
-  // it('should have the complete request feature if enabled and the post is a request', () => {
-  //   // LEJ: Need to make a window mock for this unit testing
-  //   //      context or move FEATURE_FLAGS to a "features" prop
-  //   window.FEATURE_FLAGS = { CONTRIBUTORS: 'on' }
-  //   const props = {post: {tag: 'request'}}
-  //   const wrapper = renderComponent(props)
-  //   expect(wrapper.find('Connect(CompleteRequest)').length).to.equal(1)
-  //   delete window.FEATURE_FLAGS
-  // })
+  it('should have the complete request feature if enabled and the post is a request', () => {
+    process.env.FEATURE_FLAG_CONTRIBUTORS = 'on' // window.FEATURE_FLAGS = { CONTRIBUTORS: 'on' }
```

Comment:
let's do the env changes in before and after blocks -- this way, if the test fails, the env var doesn't get deleted, and it could affect other tests